### PR TITLE
Try to make push into main branch

### PR DESCRIPTION
**Pull request by Rc561U**
Just training to create pull request

### CODE REVIEW

**Comment by Rc561U** [comment]
I don't know what it is that makes me feel like this

File: coursera.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if gigabytes_free < min_gb or percent_free < min_percent:
+        return True
+    return False
+
+def check_root_full():
+    '''Returns True is root partition is full, False otherwise'''
+    return check_disk_full(disk='/', min_gb=2, min_percent=10)
+
+def main():
+    everything_ok = True
```
Comment (VladislavRostovtsev97) [inline coursera.py]: 1.I am not sure, but it's could be true by default,  look in google
2. _ in variables name not a good practice,  better use upper and lower cases instead
3. I am would also thought about other name for this var , what is everything? Not clear for the first view.

File: coursera.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    du = shutil.disk_usage(disk)
+    #calculate the percentage of free space
+    percent_free = 100 * du.free / du.total
+    gigabytes_free = du.free / 2 **30
+    if gigabytes_free < min_gb or percent_free < min_percent:
+        return True
+    return False
+
+def check_root_full():
+    '''Returns True is root partition is full, False otherwise'''
```
Comment (VladislavRostovtsev97) [inline coursera.py]: "If root"

File: coursera.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import sys
+
+def check_reboot():
+    '''Returns True if the computer has a pending reboot'''
+    return os.path.exists('/run/reboot-required')
+
+def check_disk_full(disk, min_gb, min_percent):
```
Comment (VladislavRostovtsev97) [inline coursera.py]: Your function checks disk space and return true or false, but regarding function name it says to check only full, so I would call it checkDiskSpace

File: coursera.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    '''Returns True if there isn't enough disk space, False otherwise.'''
+    du = shutil.disk_usage(disk)
+    #calculate the percentage of free space
+    percent_free = 100 * du.free / du.total
+    gigabytes_free = du.free / 2 **30
+    if gigabytes_free < min_gb or percent_free < min_percent:
```
Comment (VladislavRostovtsev97) [inline coursera.py]: What if it equal mingb and percentage. Should be also false?

**Review by VladislavRostovtsev97 — APPROVED** [review]

**Pull request merged**